dracut-init.sh

```diff
@@ -461,6 +461,14 @@ inst_rules_wildcard() {
 
 prepare_udev_rules() {
     [ -z "$UDEVVERSION" ] && export UDEVVERSION=$(udevadm --version)
+    if [ -z "$UDEVVERSION" ]; then
+        derror "Failed to detect udev version"
+        return 1
+    fi
+    if [ -z "${UDEVVERSION##*[!0-9]*}" ]; then
+        derror "udevadm --version reported not an integer, can't handle it"
```

Comment:
Spelling: "...did not report an integer, ...".  can't -> cannot.

And "can't handle it is a bit implicit with derror, isn't it? ;-) " Maybe better:

"udevadm --version did not report an integer, udev version cannot be determined!"

(unfortunately, dracut is not consistent when it comes to punctuation here)

Reply:
Done, force-pushed.